### ISSUE TITLE
alpha 1.4 tmodloader calamity

### ISSUE THREAD

**Issue by 51nya51** [MountainDrew8/Something#27]
make calamity mod to tmodloader 1.4 alpha

**Comment by devraza** [comment]
🤦

**Comment by NestlyS** [comment]
Is there any plans for this? I think this is good idea to kill some Empress of light with Calamity weapons

**Comment by devraza** [comment]
No, there aren't any plans. And it will definitely be a long time before calamity supports 1.4. Heck, tModLoader support for 1.4 isn't finished, so you can't possible expect any big content mods to support 1.4 any time soon.

**Comment by NestlyS** [comment]
Ok, thanks anyway!
